website/project/model.py

```diff
@@ -1527,6 +1556,9 @@ def save(self, *args, **kwargs):
                 save=True,
             )
 
+            if self.creator and settings.ENABLE_NOTIFICATION_SUBSCRIPTION_CREATION:
+                self.subscribe_user_to_notifications(user=self.creator)
```

Comment:
This is OK, but it's reinventing the wheel a bit. We typically use blinker signals for this case. You could emit a signal for `node_created` here, then have a listener that created the subscription. Something like

```python
class Node(...):
    @contrib_added.connect
    @node_created.connect
    def _subscribe_user_to_notifications_listener(...):
        ...
```